docs/nodes/blockspace-race.mdx

```diff
@@ -246,6 +248,20 @@ broadcast transactions.
 * [http://blockspacerace-grpc.upnodedev.xyz/](http://blockspacerace-grpc.upnodedev.xyz/)
 * [http://celestia-itn-grpc.staking4all.org:9090/](http://celestia-itn-grpc.staking4all.org:9090/)
 * [https://grpc-blockspacerace.staker.space](https://grpc-blockspacerace.staker.space)
+* [bd-celestia-rpc-node-01.stg.bdnodes.net:443/](bd-celestia-rpc-node-01.stg.bdnodes.net:443/)
+* [bd-celestia-rpc-node-02.stg.bdnodes.net:443/](bd-celestia-rpc-node-02.stg.bdnodes.net:443/)
+
+#### Websocket endpoints
+
+The websocket endpoint is to allow users to interact with a Celestia Node using
```

Comment:
@jer117 could you please match the description with the other sections (RPC/gRPC/API)?